### DISCUSSION TITLE
SPAM TITLE

### DISCUSSION THREAD

**Discussion by Ehsanreader789** [ImageMagick/ImageMagick#2003] [General]
Removed spam post, view edit history.

**Comment by snibgo** [comment]
Yes. For example:
`magick in1.png in2.png out.gif`

**Comment by snibgo** [comment]
Evidently the OP was a sleeper post, waiting to be converted into spam. Perhaps a moderator will squish the spammer.

**Reply by dlemstra** [reply]
How did you find out that this message was edited @snibgo? Going to report this post.

**Comment by snibgo** [comment]
My reply on 10 May 2020 was to what seemed like a genuine question. That question can be viewed by clicking on "edited" top-right of the post. The same "edited" button shows the original poster edited the post 11 hours ago, by removing the original text and inserting the new text.

**Reply by dlemstra** [reply]
I know what happened :) Was just wondering how you found out that they edited this post.

**Comment by snibgo** [comment]
I clicked on "edited".

**Reply by dlemstra** [reply]
Sorry for all the question but I wonder how you found out this post was edited.
